src/roadmap/config.py

```diff
@@ -19,6 +22,7 @@ class Settings(BaseSettings):
     debug: bool = False
     dev: bool = False
     host_inventory_url: str = "https://console.redhat.com"
+    upcoming_json_path: FilePath = str(Path(__file__).parent.joinpath("data").resolve().joinpath("upcoming.json"))
```

Comment:
Casting this to a `str` removes all the nice functionality the `pathlib` provides.

Keep this path relative.
```suggestion
    upcoming_json_path: FilePath = Path(__file__).parent.joinpath("data").joinpath("upcoming.json")
```

Reply:
I did the `str` because pydantic was giving me a warning about an implicit conversion to string, but I can't reproduce the warning.